From: Jinoh Kang <jinoh.kang.kr@gmail.com>
Subject: [PATCH] include: Remove duplicate definitions of UOI_* macros.
Message-Id: <7599a027-4d7e-87d8-5153-e3827e53d2ee@gmail.com>
Date: Sun, 17 Apr 2022 00:24:42 +0900

Commit 5ec458fd41f (Add declarations for GetClassLongPtr{A,W}, GCLP_*
defines and user object information defines., 2004-10-14) inadvertently
introduced duplicate definitions of UOI_* macros.

While multiple identical definitions of a macro do not cause compilation
problems, deduplicating them faciliates future expansion of UOI_*
constants.

Signed-off-by: Jinoh Kang <jinoh.kang.kr@gmail.com>
---
 include/winuser.h | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/include/winuser.h b/include/winuser.h
index 11111111111..11111111111 100644
--- a/include/winuser.h
+++ b/include/winuser.h
@@ -2874,12 +2874,6 @@ typedef struct {
 #define	DSS_PREFIXONLY	0x0400
 #define	DSS_RIGHT	0x8000
 
-/* UserObjectInformation classes */
-#define UOI_FLAGS 1
-#define UOI_NAME 2
-#define UOI_TYPE 3
-#define UOI_USER_SID 4
-
 
 /* Sent as the lParam of a WM_DRAWITEM message to instruct how an
  * owner drawn control is to be drawn */

-- 
2.35.3

